includes/Lifecycle.php

```diff
@@ -307,4 +308,18 @@ protected function upgrade_to_2_5_0() {
 		 */
 		as_unschedule_all_actions( Products\Feed::GENERATE_FEED_ACTION );
 	}
+
+	protected function upgrade_to_3_1_13() {
+		$notice_slug          = 'facebook_messenger_deprecation_warning';
+		$is_messenger_enabled = get_option( \WC_Facebookcommerce_Integration::SETTING_ENABLE_MESSENGER, 'no' ) === 'yes';
```

Comment:
I don't think we need the \ for internal these classes. And it used inconsistently throughout

Reply:
Ah, good point. I think this was just lifted from https://github.com/woocommerce/facebook-for-woocommerce/blob/7f35cdefb121d1e0d33371ce2c0a7e94980405a6/includes/Admin/Settings_Screens/Messenger.php#L128-L129

Reply:
Seems like it's actually pretty widely used: https://github.com/search?q=repo%3Awoocommerce%2Ffacebook-for-woocommerce+WC_Facebookcommerce_Integration%3A%3A&type=code

And it throws an error without:
![image](https://github.com/woocommerce/facebook-for-woocommerce/assets/228780/091be0f9-8fca-4f50-873d-11b05bd995cb)

Reply:
that's surprising. thanks for looking into it